composer.json

```diff
@@ -9,35 +9,43 @@
       "email": "wearnhardt@gmail.com"
     }
   ],
-  "repositories": [
-    {
+  "repositories": {
+    "newfold": {
       "type": "composer",
-      "url": "https://bluehost.github.io/satis/"
+      "url": "https://newfold-labs.github.io/satis/",
+      "only": [
+        "newfold-labs/*"
+      ]
     }
-  ],
+  },
   "autoload": {
     "psr-4": {
       "Bluehost\\Maestro\\": "inc/"
     }
   },
   "require": {
-    "firebase/php-jwt": "^5.0"
+    "newfold-labs/wp-module-maestro": "@dev"
```

Comment:
@amartya-dev We'll want to make sure we change from `@dev` to `^1.0`, which means we'll need to tag a `1.0.0` version of the module first.

Reply:
Done, updated the version